prime-router/src/test/kotlin/azure/ActionHistoryTests.kt

```diff
@@ -222,7 +208,7 @@ class ActionHistoryTests {
      * What I'd really like to do is confirm that two sql inserts were generated,
      * one to insert into ACTION and one to insert into REPORT_FILE.
      */
-//    @Test
+    @Test @Disabled
```

Comment:
Is this disabling intended? Why? Can you explain the reason for the disabling (maybe in the code with a `// TODO, #ISSUE_NUMBER` next to it for future re-enabling)

Reply:
I added disabled because someone else commented out the @Test.  I cannot explain why it was done, but the proper way is to use @Disabled.